test/UnitTimeoutNone.cpp

```diff
@@ -31,9 +31,12 @@ static constexpr size_t ConnectionCount = 9;
 /// Base test suite class for connection limit (no limits) using HTTP and WS sessions.
 class UnitTimeoutNone : public UnitTimeoutBase1
 {
-    void configure(Poco::Util::LayeredConfiguration& /* config */) override
+    void configure(Poco::Util::LayeredConfiguration& config) override
     {
         // Keep original values -> No timeout
+
+        // Disable `net::Defaults.maxExtConnections` instrument
+        config.setInt64("net.max_ext_connections", -1);
```

Comment:
Explained in NetUtils.hpp for `net::Defaults.maxExtConnections`
>     /// Maximum number of concurrent external TCP connections. Zero disables instrument.
>+    /// Set to COOLWSD config `net.max_ext_connections` if >= `COOLWSD::MinConnectedSessions`
>+    /// or is disabled (zeroed) if config < 0.
>+    /// Otherwise set to system-value, see `Util::getMaxConcurrentTCPConnections`.
>     size_t maxExtConnections;

So the tri-state: <0 disable, [0-2] ignore for system defaults and >=3 is required to fit all needs IMHO.
Differentiation of config value semantics and `net::Defaults.maxExtConnections`

Reply:
The comment on line 38 of test/UnitTimeoutNone.cpp says "Disable" and sets the value to -1. 
The comment on line 75 of net/Socket.cpp says "disable" and sets the value to 0.

Perhaps the issue is the comments?

Reply:
Yes, I also changed the comment in that use-case, since the config value (tri-state) and the stored value (off or number) in net::Defaults do have different semantics. 
I was indeed teared apart by commenting on this in more detail, but then removed it in favor of the single comment in net::Defaults/DefaultValue. 
`        // Disable `net::Defaults.maxExtConnections` instrument, i.e. setting it to zero (skipping system default)`